types/webgl2/index.d.ts

```diff
@@ -407,6 +425,12 @@ interface WebGL2RenderingContext extends WebGLRenderingContext {
     compressedTexSubImage3D(target: number, level: number, xoffset: number, yoffset: number,
         zoffset: number, width: number, height: number, depth: number,
         format: number, imageSize: number, offset: number): void;
+    compressedTexSubImage3D(target: number, level: number, xoffset: number, yoffset: number,
```

Comment:
`compressedTexSubImage3D` was also introduced in WebGL2, so it shouldn't clash with anything in lib.dom.d.ts right now.

And here as well: If you want to keep the verbose version, please remove `null`. The [specification](https://www.khronos.org/registry/webgl/specs/latest/2.0/) states that `srcData` has to be an `ArrayBufferView` (which excludes `null`).